src/main/webapp/i18n/de/exam.json

```diff
@@ -690,6 +690,10 @@
                 "actions": "Aktionen",
                 "viewResultsOrRunDetection": "Ergebnisse anzeigen oder Plagiatsprüfung starten",
                 "viewCases": "Fälle anzeigen"
+            },
+            "dateChange": {
+                "title": "Die Änderung der Zeiten einer aktiven Klausur wird den Arbeitsablauf der Studierenden stören!",
```

Comment:
```suggestion
                "title": "Eine Änderung der Arbeitszeiten einer aktiven Klausur wird den Arbeitsablauf der Studierenden stören!",
```

Reply:
Or probably

```suggestion
                "title": "Eine Änderung der Arbeitszeit einer aktiven Klausur wird den Arbeitsablauf der Studierenden stören!",
```

Reply:
Changed it altogether now, shortening the title, as it's very long for the modal. The body text will contain the information.